smart-contracts/test/Unlock/uniswapValue.mainnet.js

```diff
@@ -44,7 +44,7 @@ contract('Unlock / uniswapValue', () => {
     unlock = await ethers.getContractAt('Unlock', unlockAddress)
 
     // deploy oracle
-    oracle = await deployUniswapV3Oracle()
+    oracle = await deployUniswapOracleV3()
```

Comment:
Why the change?

Reply:
was a typo in the name I discovered when running the test but will fix in the lib to keep `UniswapV3` in the name.